official/resnet/estimator_benchmark.py

```diff
@@ -342,6 +350,63 @@ def __init__(self, output_dir=None, root_data_dir=None, **kwargs):
         output_dir=output_dir, default_flags=def_flags)
 
 
+class Resnet50MultiWorkerEstimatorBenchmark(Resnet50EstimatorBenchmarkBase):
+  """Benchmarks for ResNet50 using Estimator with multiple workers."""
+
+  def __init__(self, output_dir=None, default_flags=None):
+    super(Resnet50MultiWorkerEstimatorBenchmark, self).__init__(
+        output_dir=output_dir,
+        default_flags=default_flags)
+
+  def benchmark_graph_fp16_8_gpu_ring_tweaked(self):
```

Comment:
How do we know how many workers and total GPUs used?

Reply:
The benchmark expects `TF_CONFIG` environment variable to be set with the list of workers and their IP addresses.